src/number/n_unknown.jl

```diff
@@ -509,30 +509,67 @@ end # for (rw, rew) in
 
 CoefficientRingID = Dict{Nemo.Ring, Any}()
 
-# FIXME: We need caching here? And the caching mechanism is not type stable?
-function CoefficientRing(R::Nemo.Ring, cached::Bool = true)
+if VERSION >= v"1.7"
+  _can_i_haz_mutable(::Type{T}) where {T} = ismutabletype(T) && ismutabletype(elem_type(T))
```

Comment:
Clearly this function name is indeed the cr0wn of perfection, so there is indeed no point in discussing it any further 😂

Reply:
(Though personally I would just do
```julia
if VERSION < v"1.7"
   ismutabletype(::Type{T}) where T = T.mutable
end
```